Dockerfile

```diff
@@ -1,18 +1,23 @@
 FROM alpine:edge
 
+ENV K8S_VERSION=v1.6.3
```

Comment:
tbh backwards compatibility has always been good with kubectl, I almost always run the latest version on my workstation and I have never had issues, so yes...